### ISSUE TITLE
Penumbra Collection Swap not working

### ISSUE THREAD

**Issue by GunB** [PayneZA/OutfitManager#2]
I'm testing the plugin and is really helpful to change outfits and character looks via chat macros. Seems Penumbra collections are not being changed properly. I tried with a name without spaces and also an invalid name. 
The invalid name showed correctly the collection didn't exist. For the other options I tried, the collection would simply not be applied.

**Comment by PayneZA** [comment]
Can you try seeing what using the penumbra command does, I have put it below, just change out the obvious "REPLACE_WITH_YOUR_COLLECTION_NAME" to the collection thats not working out. It only caters for the "Your Character" being the only collection setup.

/penumbra collection Your Character | REPLACE_WITH_YOUR_COLLECTION_NAME | p | yourself

**Comment by GunB** [comment]
Doesn't seem to change it. I only changed REPLACE_WITH_YOUR_COLLECTION_NAME with teh collection name and it says the collection was applied but nothing changed. Do I need to enable something for this to work?

**Comment by GunB** [comment]
I used the following command and it did work

/penumbra collection Individual | REPLACE_WITH_YOUR_COLLECTION_NAME  | <me>

**Comment by PayneZA** [comment]
I have made the change for it to make use of <me> (which may help) however for your own character you should be using the Character Groups option in penumbra, one of the options there is "Your Character", that is where it changes the collection.

That part I cannot change as its how penumbra recommend it in their official help below.
https://reniguide.info/#specialcollections

**Comment by GunB** [comment]
I don't like using the "Your Character" option because I have multiple alts which I love having the option to make them look like whatever I want. With the "Individual" option, It can support multiple characters without messing other's collections.

Having the option to quick-change by using the < me > option would really give it a very powerful usage instead of being lilmited to the name written in the plugin.

Right now I'm having issues with glamourer too. It says it can't find a character name I'm not logged in at that moment.

The macro I'm using currently is:

```
/penumbra collection Individual | XX | <me>
/glamour Apply,<me>,My Chars/XX
/penumbra redraw <me>
```

And this works wonders except it doesn't persist. 
Not sure if this was the intended usage or it would even be possible to support it but I really appreciate the support and quick response.

**Comment by PayneZA** [comment]
I have made the change, I did not put changing the collection type in the main gui to not confuse people however there is now a chat command to change away to whichever you wish.

/omg setcollectiontype Individual for example or /omg setcollectiontype reset (to go back to default).
glamourer also uses <me> now

I dont use redraw since that happens on applying the glamourer design.

Thankyou for your helpful feedback, multiple character support should now be in effect as well as not re-applying the collection if the person wishes.
